### DISCUSSION TITLE
关于kKrakenJSBridgePoolSize = 8的问题

### DISCUSSION THREAD

**Discussion by donfo** [openkraken/kraken#934] [Q&A] [answered]
[kraken/lib/src/bridge/bridge.dart#L16](https://github.com/openkraken/kraken/blob/94abb6ba41e2cfe61ea9de447919d1d8abfdeda2/kraken/lib/src/bridge/bridge.dart?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L16)
```dart
/// the Kraken JS Bridge Size
int kKrakenJSBridgePoolSize = 8;
```
@andycall 大佬，请问当前对KrakenJSBridge为8的限制做调整会产生什么影响吗？

我们当前的业务场景大量使用了Kraken，实际场景会超过8这个数字。比如APP底部会有多个BottomNavigationBar，BottomNavigationBar下面又会有多个子的TabBar，TabBar通过动态配置的Kraken的bundleURL。

**Comment by andycall** [comment, marked as answer]
没有限制，可以通过覆盖这个值来调整

https://github.com/openkraken/kraken/blob/main/kraken/lib/src/bridge/bridge.dart#L16
